src/database/Database.cpp

```diff
@@ -119,6 +119,7 @@ Database::applySchemaUpgrade(unsigned long vers)
 
     case 4:
         BanManager::dropAll(*this);
+        DataFrame::dropAll(*this); // not and error, it changed again
```

Comment:
damn I didn't see lastmodified in the xdr. This is a feature flag then since the older ledgers need to keep lastmodified as 0. 

SCHEMA_VERSION was rev'd. we are bundling several changes into this branch